src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage = () => {
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    getPeople()
+      .then(setPeople)
+      .catch(() => {
+        setIsError(true);
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, []);
+
+  const everythingOk = !isError && !isLoading && people.length > 0;
+  const isEmpty = !isError && !isLoading && people.length === 0;
```

Comment:
What exactly is empty?
```suggestion
  const isEmpty = !isError && !isLoading && !people.length;
```